aws/src/database/dynamodb/scripts/asset_table/patch_asset_with_additional_boiler_house_id.py

```diff
@@ -0,0 +1,59 @@
+from dataclasses import dataclass
+
+from mypy_boto3_dynamodb.service_resource import Table
+
+from aws.src.database.dynamodb.utils.get_by_secondary_index import get_by_secondary_index
+from aws.src.database.dynamodb.utils.get_dynamodb_table import get_dynamodb_table
+from aws.src.utils.csv_to_dict_list import csv_to_dict_list
+from aws.src.utils.logger import Logger
+from aws.src.utils.progress_bar import ProgressBar
+from enums.enums import Stage
+from decimal import Decimal
+
+
+@dataclass
+class Config:
+    TABLE_NAME = "Assets"
+    LOGGER = Logger()
+    STAGE = Stage.HOUSING_PRODUCTION
```

Comment:
not staging first?

Reply:
I have since run this against staging. The script is run locally, so the stage can be changed to any environment. Im pushing the script to the branch for future reference